### ISSUE TITLE
Instagram Embed Block 

### ISSUE THREAD

**Issue by ascordato** [WordPress/gutenberg#3499]
## Issue Overview
The embed block for Instagram doesn't work. 

Currently running latest version of WP with Gutenberg 1.7 installed. Reproduced this error in Chrome and Safari. 

## Steps to Reproduce (for bugs)

1. Add the block to a post
2. Paste the Instagram URL for the image 

I also tried pasting the actual embed code Instagram provides too and the message "Please enter a URL" appears. 

## Current Behavior
![screen shot 2017-11-15 at 1 28 54 pm](https://user-images.githubusercontent.com/6915383/32855729-96ab8f24-ca10-11e7-813c-540981e2b29f.png)

## Todos
- [ ] Tests
- [ ] Documentation

**Comment by aduth** [comment]
Can you share the URL you're using for the embed? I am not able to recreate the issue with a sample URL:

https://www.instagram.com/p/Bbba7EYljLv/

**Comment by ascordato** [comment]
Here you go: https://www.instagram.com/p/Bbb011mBv5K/ 

I tried the URL you provided above and got the same error.

**Comment by gziolo** [comment]
I was able to reproduce on ver. 1.7

**Comment by hedgefield** [comment]
I was able to embed both URLs successfully on 1.7. They do seem kinda squished horizontally though, but that's another design matter.

**Comment by ascordato** [comment]
I just tried again on my end after upgrading to 4.9 and still get the same
error. FWIW, I can get other embed blocks like Youtube to work as expected.

**Comment by Soean** [comment]
I was able to embed Instagram posts with gutenberg 1.7.

![instagram](https://user-images.githubusercontent.com/695201/32950797-f648e394-cba7-11e7-95ab-93e9fbdd51d2.png)

**Comment by GaryJones** [comment]
With GB 1.9.0, it works, but only appears as about 50% width of the content area. Tried with a video and photo.

![screenshot 2017-12-12 12 50 57](https://user-images.githubusercontent.com/88371/33885421-302df51c-df3b-11e7-970b-7f829c5af72e.png)

Aside: The Flickr Embed also seems to suffer from the same symptoms - can't expand to be the width of the rest of the textual content, can't center it within the content area etc.

**Comment by clintlalonde** [comment]
Having same issue. Getting error when trying to embed an Instagram photo. Using my own photo (https://www.instagram.com/p/BX-4fN9Dkcy/) and the test photo given above (https://www.instagram.com/p/Bbba7EYljLv/) gets same results. 
![image](https://user-images.githubusercontent.com/679506/34500307-0dacc632-efbf-11e7-984d-b6ec01c8054c.png)

On self-hosted WordPress 4.9.1 using Guttenberg 1.9.1

**Comment by abdullah1908** [comment]
Facing the same issue and tried different Instagram profile URLs but the same result
![instagram issue](https://user-images.githubusercontent.com/14175927/36521026-573ba416-17b6-11e8-9e94-3e75d880e767.png)
WordPress _4.9.4_
Gutenberg _2.2_
Google Chrome _64.0.3282.167_

**Comment by aaronjorbin** [comment]
I ran into this issue today.  

WordPress: nightly
Guteneberg 2.6: https://www.instagram.com/p/BhR3XVRFSJL/
Chrome: 65.0.3325.181 

Instagram URL trying to embed: https://www.instagram.com/p/BhR3XVRFSJL/


I've tracked this down to being an issue caused by jetpack.  It's removal of the Instagram provider in `jetpack/modules/shortcodes/instagram.php` means that it isn't being picked up by the proxy.
